src/apiCalls.js

```diff
@@ -0,0 +1,18 @@
+const fetchMovies = {
+    allMovieData(){
+        return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+            .then(response => {
+                if(!response.ok) {
+                    throw new Error("Unable to retrieve data")
+                } else {
+                    return response.json()
+                }
+            })
+    }
+    // ,
```

Comment:
Before making a PR, be sure to remove any commented-out code. Delete it - you can do it!